packages/eden/src/client.ts

```diff
@@ -93,32 +93,32 @@ export class EdenClient<TElysia extends AnyElysia = AnyElysia> {
     return abortablePromise
   }
 
-  public query(params: EdenRequestParams, options?: EdenClientRequestOptions) {
+  public query(parameters: EdenRequestParameters, options?: EdenClientRequestOptions) {
     return this.promisifyRequest<any, EdenResponse>({
       type: 'query',
-      params,
+      params: parameters,
```

Comment:
Weird inconsistency; if we want to use `parameters` as the designated name, then it should be the case for this property name too.

Same applies for the other similar calls.